docs/standalone.md

```diff
@@ -1,3 +1,240 @@
+
+``` 
+$ git clone https://github.com/kubernetes/heapster.git
+```
+
+**Step 2: Build the source code**
+
+``` 
+$ cd kubernetes/heapster
+$ godep go build
+```
+
+Then you can get the heapster binary in current path.
+
+### Running a cAdvisor
+cAdvisor can run in a docker or standalone outside of Docker to monitor the whole machine. This guide run a cAdvisor standalone out of Docker.cAdvisor is a static Go binary with no external dependencies. To run it standalone all you should need to do is run it! Note that some data sources may require root priviledges.You can get cAdvisor binary from the [Release Page](https://github.com/google/cadvisor/releases)
+
+``` 
+$ ./cadvisor  
+```
+
+cAdvisor is now running (in the foreground) on http://localhost:8080/.
+
+## Starting a heapster
+
+### Config Cadvisor source
+Heapster support two types of Cadvisor source: standalone & CoreOS; Doc of source configuration is [HERE](https://github.com/kubernetes/heapster/blob/master/docs/source-configuration.md)
+
+External cadvisor source "discovers" hosts from the specified file. Use it like this:
+
+```
+--source=cadvisor:external[?<OPTIONS>]
+```
+
+The following options are available:
+
+* `standalone` - only use `localhost` (default: `false`)
+* `hostsFile` - file containing list of hosts to gather cadvisor metrics from (default: `/var/run/heapster/hosts`)
+* `cadvisorPort` - cadvisor port to use (default: `8080`)
+
+Here is an example:
+```shell
+$ ./heapster --source="cadvisor:external?cadvisorPort=4194"
+```
+
+Because heapster needs a specified file to config external cadvisor source, we need to provide one.
+The `hostsFile` parameter defines a list of hosts to poll for metrics and must be in JSON format. See below for an example:
+
+```
+{
+  "Items": [
+    {
+      "Name": "server-105",
+      "IP": "192.168.99.105"
+    },
+    {
+      "Name": "server-106",
+      "IP": "192.168.99.105"
+    }
+  ]
+}
+```
+### Start heapster
+Now you can run a heapster, here is an example:
+
+```
+./heapster-master --source="cadvisor:external?cadvisorPort=8080" --use_model=true --model_resolution=10s --cache_duration=1m --poll_duration=2s --stats_resolution=1s"
+```
+
+The `hostsFile` use the default one `/var/run/heapster/hosts`.Then the heapster is running on the default port 8082,like this:
+
+```
+I0911 13:42:03.657739   21041 heapster.go:57] ./heapster --source=cadvisor:external?cadvisorPort=8080 --use_model=true --model_resolution=10s --cache_duration=1m --poll_duration=2s --stats_resolution=1s
+I0911 13:42:03.657918   21041 heapster.go:58] Heapster version 0.17.0
+I0911 13:42:03.662594   21041 heapster.go:68] Starting heapster on port 8082
+```
+
+###Test it out
+If you got debug information as before, means that the heapster is working normally.You can test it by requet its [Reatful api](https://github.com/kubernetes/heapster/blob/master/docs/model.md). Like this:
+
+```
+$ curl http://localhost:8082/api/v1/model/stats/
```

Comment:
Not sure about your curl but my (7.35.0 (x86_64-pc-linux-gnu)) doesn't automatically gunzip the response.
Response headers: 
```
> GET /api/v1/model/stats/ HTTP/1.1
> User-Agent: curl/7.35.0
> Host: localhost:8082
> Accept: */*
> 
< HTTP/1.1 200 OK
< Content-Encoding: gzip
< Content-Type: application/json
< Date: Fri, 11 Sep 2015 09:35:21 GMT
< Content-Length: 625
```
Response content:
```
��_��WA��q0�s�P3�[...]
```